web/models.py

```
@@ -133,6 +133,54 @@
+    def __str__(self):
+        return self.text
+
+
+class Response(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    question = models.ForeignKey("Question", on_delete=models.CASCADE)
+    choice = models.ForeignKey("Choice", on_delete=models.CASCADE, blank=True, null=True)
+    text_answer = models.TextField(blank=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)